### PR TITLE
prometheus-cups-exporter: This shouldn't be packaged

### PR DESCRIPTION
**Pull request by ThoreKr**
This exporter is outdated, broken by design and simply shouldn't be packaged.

###### Motivation for this change
There is no point in having this garbage version as a package. For peace of mind I want it gone.

###### Things done
Deleted the according .nix file created in #67634

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cole-h** [comment]
Some comments:

1. You need to remove it from `pkgs/top-level/all-packages.nix`.
2. You should add an alias to `pkgs/top-level/aliases.nix` stating why it was removed.
3. If it were me, I'd reword the commit to be `prometheus-cups-exporter: remove` or `...: drop`.

**Comment by ThoreKr** [comment]
@cole-h Thanks for the remarks, I fixed these points.

**Review by cole-h — APPROVED** [review]
Perfect, thank you!
